applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,337 @@
+
+Firmware architecture
+=====================
+
+The nRF Peripheral Sensor node application has a modular structure, where each module has a defined scope of responsibilities.
+The application uses the :ref:`app_event_manager` to distribute events between modules in the system.
+
+The following figure shows the application architecture.
+The figure visualizes relations between Application Event Manager, modules, drivers, and libraries.
+
+.. figure:: /images/peripheral_sensor_node_arch.svg
+   :alt: Peripheral Sensor node application architecture
+
+   Peripheral Sensor node application architecture
+
+Programming nRF54L15 PDK
+========================
+
+The nRF54L15 PDK has a J-Link debug IC that can be used to program the firmware.
+
+See the `Building and running`_ section for more information.
+
+Requirements
+************
+
+The application supports the following development kit:
+
+.. table-from-sample-yaml::
+
+User interface
+**************
+
+You can interact with the application by using the logging system and Bluetooth application described in the :ref:`nrf_peripheral_sensor_node_app_testing` section.
+
+Configuration
+*************
+
+The nRF Peripheral Sensor node application is modular and event-driven.
+You can enable and configure the modules separately for the selected board and build type.
+See the documentation page of the selected module for information about functionalities provided by the module and its configuration.
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options:
+
+.. _CONFIG_APP_CONTROL_BURST_INTERVAL_S:
+
+CONFIG_APP_CONTROL_BURST_INTERVAL_S
+   The application configuration specifies the time in seconds between burst data acquisitions from sensors.
+   If sensor data burst is not triggered by ``wake_up_event`` from IMU, burst is performed every interval value.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_CNT:
+
+CONFIG_APP_CONTROL_BURST_MEAS_CNT
+   The application configuration specifies the number of measurements a single burst acquires before going to sleep.
+   The default value is ``10``.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS:
+
+CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS
+   The application configuration specifies the time interval between measurements for a single burst.
+   The default value is 1000 ms.
+
+Configuration files
+===================
+
+The nRF Peripheral Sensor node application uses the following files as configuration sources:
+
+* Devicetree Specification (DTS) files - These reflect the hardware configuration.
+  See :ref:`zephyr:dt-guide` for more information about the DTS data structure.
+* Kconfig files - These reflect the software configuration.
+  See :ref:`kconfig_tips_and_tricks` for information about how to configure them.
+* :file:`_def` files - These contain configuration arrays for the application modules.
+  The :file:`_def` files are used by the nRF Peripheral Sensor node application modules and :ref:`lib_caf` modules.
```

Comment:
Can we link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/config_and_build_system.html instead of listing these? This is mostly basic knowledge for Zephyr and NCS users.
```suggestion
The nRF Peripheral Sensor node application uses both the :ref:`standard configuration files common to Zephyr and the |NCS| <app_build_system>` and the application-specific :file:`_def` files, which contain configuration arrays for the application modules. The :file:`_def` files are used by the nRF Peripheral Sensor node application modules and :ref:`lib_caf` modules.
```